src/components/NavBar/Navbar.css.ts

```diff
@@ -0,0 +1,34 @@
+import { style } from '@vanilla-extract/css'
```

Comment:
We should be using `styled-components` here since this is shared code. I think using `vanilla-extract` for NFT-specific pages is okay since it's isolated and we're punting on deciding whether to migrate or not.

Reply:
I disagree here. Seeing as they're both just css in js solutions there's no functionality differences. NFT devs are more comfortable using the design system we've already been using, but if other engs want to re-write it in styled components I wouldn't be opposed.